src/pytest_regressions/common.py

```diff
@@ -188,3 +189,24 @@ def make_location_message(banner: str, filename: Path, aux_files: List[str]) ->
             else:
                 dump_aux_fn(Path(obtained_filename))
                 raise
+
+
+def round_digits(data: Union[list, dict], prescision: int) -> Union[list, dict]:
```

Comment:
Let's use the generic types instead of concrete ones, as they encompass more types, and a TypeVar to properly type the return value.


```suggestion
T = TypeVar("T", bound=Union[MutableSequence, MutableMapping])

def round_digits(data: T, precision: int) -> T:
```